src/wallet/sendTransaction.ts

```diff
@@ -166,8 +167,10 @@ class SendTransactionWalletService extends EventEmitter {
         throw new SendTxError(`Address ${o.address} is not valid.`);
       }
       const tokenData = (o.token in tokens) ? tokens.indexOf(o.token) + 1 : 0;
-      const outputOptions = { tokenData, timelock: o.timelock || null };
-      outputsObj.push(new Output(o.value, address, outputOptions));
+      const outputOptions = { tokenData };
+      const p2pkh = new P2PKH(address, { timelock: o.timelock || null });
+      const p2pkhScript = p2pkh.createScript()
+      outputsObj.push(new Output(o.value, p2pkhScript, outputOptions));
```

Comment:
Maybe `p2pkh.createOutput()`?

Reply:
Do you mean to create a method in P2PKH class that receives value and tokenData and creates an output from a P2PKH object?

Reply:
As we've discussed in a call there is no need to create this method, it wouldn't help much.